src/config.ts

```diff
@@ -109,6 +110,10 @@ const config: Config = {
         message_id: "841608870001508372",
       },
     ],
+    excludedChannels: [
+      '748546478825275435',
```

Comment:
Can we add the channel names as an `// inline-comment` next to it, so we know what channels the id's represent while reading the code?